tests/integrations/test_monkeypatch_keras.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
+def test_import_order():
+    # monkeypatching tf.keras caused import issue
+    WandbCallback = pytest.importorskip(
+        "wandb.keras.WandbCallback", reason="imports tensorflow"
+    )
+
+    tf = pytest.importorskip(
+        "tensorflow", minversion="2.6.2", reason="only relevant for tf>=2.6"
+    )
+    keras = pytest.importorskip(
+        "keras", minversion="2.6", reason="only relevant for keras>=2.6"
+    )
+
+    assert isinstance(tf.keras.Model(), keras.Model)
```

Comment:
Does this work when someone imports keras before tf and vice versa in their script? Should we have a test showing the converse/altnerate import orders to keep it from breaking?

Reply:
oh yeah... the issue was only our import order i.e. `wandb` before `tf`/`keras`. 
the order between `tf` and `keras` doesn't matter. but i can add more permutations if you think it's helpful